storage/myisam/ha_myisam.cc

```diff
@@ -194,6 +194,59 @@ static void mi_check_print_msg(HA_CHECK *param,	const char* msg_type,
 
 
 /*
+ stores information about key parts.
+
+  SYNOPSIS
+    setup_keyparts()
+      key_info    in     Key definitions.
+      keyseg      in     key segment.
+      table_arg   in     TABLE object.
+      key_part_no in    index of key part.
+
+ */
+void setup_keyparts(KEY* key_info, HA_KEYSEG *keyseg, TABLE *table_arg, uint key_part_no)
+{
+	enum ha_base_keytype type= HA_KEYTYPE_BINARY;
+	Field *field= key_info->key_part[key_part_no].field;
+	type = field->key_type();
+	keyseg->flag= key_info->key_part[key_part_no].key_part_flag;
+	keyseg->type= (int)type;
+  keyseg->start= key_info->key_part[key_part_no].offset;
+	keyseg->length= key_info->key_part[key_part_no].length;
```

Comment:
Use same style as rest of the file, 2 spaces and make sure that every line is indented similarly.